x-pack/filebeat/input/entityanalytics/provider/okta/internal/okta/ratelimiter_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"time"
 
 	"github.com/elastic/elastic-agent-libs/logp"
+	"golang.org/x/time/rate"
```

Comment:
Needs goimports or manual separation.

Reply:
This was less automatic than I expected. Fixed.